src/FormatNumber/Parser.elm

```diff
@@ -378,3 +442,54 @@ parse locale original =
                 | prefix = locale.zeroPrefix
                 , suffix = locale.zeroSuffix
             }
+
+
+{-| Given a `Locale` parses a `String` into a `Maybe Float`
+-}
```

Comment:
We need some examples here, both for documentation and for tests ; )

Reply:
Reason I did not add any tests here is because parse function in FormatNumber.elm is just calls this function and does nothing else. I thought it would be repetition.

Reply:
> parse function in FormatNumber.elm is just calls this function

So, what about moving this function to `FormatNumber.elm`?